csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -56,7 +81,47 @@ public sealed class OutgoingRequestFrame : OutgoingFrame
         // _defaultBinaryContext is empty.
         private readonly bool _writeSlot0;
 
-        /// <summary>Create a new OutgoingRequestFrame.</summary>
+        /// <summary>Creates an <see cref="OutgoingRequestFrame"/> factory for the given parameterless operation.
+        /// </summary>
+        /// <param name="operationName">The operation name.</param>
+        /// <param name="idempotent"><c>True</c> if the requests are idempotent, <c>False</c> otherwise.</param>
+        public static Factory CreateFactory(string operationName, bool idempotent) =>
+            (prx, context) => WithEmptyParamList(prx, operationName, idempotent, context);
+
+        /// <summary>Creates an <see cref="OutgoingRequestFrame"/> factory for the given operation.</summary>
+        /// <param name="operationName">The operation name.</param>
+        /// <param name="idempotent"><c>True</c> if the requests are idempotent, <c>False</c> otherwise.</param>
+        /// <param name="compress"><c>True</c> if the request's parameters are compressed during frame creation with
+        /// ice2, or if entire request frame is compressed during sending with ice1; otherwise, <c>False</c>.</param>
+        /// <param name="format">The format for class instances.</param>
+        /// <param name="writer">The <see cref="OutputStream"/> writer used to write the request parameters.</param>
+        public static Factory<T> CreateFactory<T>(
+            string operationName,
+            bool idempotent,
+            bool compress,
+            FormatType format,
+            OutputStreamValueWriter<T> writer)
+            where T : struct =>
+            (IObjectPrx prx, in T paramList, IReadOnlyDictionary<string, string>? context) =>
+            WithParamList(prx, operationName, idempotent, compress, format, context, in paramList, writer);
+
+        /// <summary>Creates an <see cref="OutgoingRequestFrame"/>for the given single-parameter operation.</summary>
```

Comment:
Missing "factory" in "Creates an ... for"

Reply:
Fixed.